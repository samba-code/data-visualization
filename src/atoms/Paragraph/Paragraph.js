import React from "react";
import styled from "styled-components";
import { rem } from "polished";

import PropTypes from "prop-types";

const StyledPara = styled.p`
  font-size: ${rem("18px")};
  line-height: ${rem("22px")};
  color: ${(props) => props.theme.primary};
  margin: 20px 0;
  max-width: 620px;
  font-family: "museo-sans", sans-serif;
  font-weight: 300;
`;

const Paragraph = ({ children }) => <StyledPara>{children}</StyledPara>;

Paragraph.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
  ]).isRequired,
};

export default Paragraph;
