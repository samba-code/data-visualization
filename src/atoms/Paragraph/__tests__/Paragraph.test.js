import React from "react";
import { render } from "enzyme";

import Paragraph from "../Paragraph";

describe("Paragraph", () => {
  it("should match snapshot", () => {
    const wrapper = render(<Paragraph>Some text</Paragraph>);
    expect(wrapper).toMatchSnapshot();
  });
});
