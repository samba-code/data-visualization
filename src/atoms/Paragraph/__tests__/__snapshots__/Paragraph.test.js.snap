// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should match snapshot 1`] = `
<p
  class="Paragraph__StyledPara-teknvt-0 eCRiac"
>
  Some text
</p>
`;
